docs/scrnaseq/README.md

```diff
@@ -142,7 +142,7 @@ The specific type of assay being executed.
 
 | constraint | value |
 | --- | --- |
-| enum | `scRNAseq-10xGenomics`, `scRNAseq`, `sciRNAseq`, `snRNAseq`, or `SNARE2-RNAseq` |
+| enum | `scATAC-seq (SNARE-seq) [Lab Processed]`, `scATAC-seq (SNARE-seq) [SnapATAC]`, `scRNA-seq (10x Genomics)`, `scRNA-seq (10x Genomics) [Salmon]`, `scRNA-seq (SNARE-seq) [Lab Processed]`, or `scRNA-seq (SNARE-seq) [Salmon]` |
```

Comment:
Again, really not sure this is correct.

Reply:
@mccalluc NOT CORRECT!
sc and sn are not the same. 

sc = single cell.
sn = single nucleus.

Reply:
"Lab processed" and "Salmon" are not an assay types. These terms refer to how the data generated by the assay is processed.

Reply:
I haven't heard of Snap ATAC.

Reply:
> sc and sn are not the same.

Green is the new version... and it is all `sc`: The old version (red) contained both `sc` and `sn`.